src/roboat_ros_node.py

```diff
@@ -46,14 +48,14 @@ def __init__(self, node_name, visual_node_name):
       mrk = Marker()
       mrk.header.frame_id = 'odom'
       mrk.action = Marker.ADD
-      mrk.lifetime.secs = 1.0
+      mrk.lifetime.secs = int(1.0)
```

Comment:
Otherwise the script crashes, shouldn't change much in the result of the script.